### DISCUSSION TITLE
telescope.lua keymaps appear to be ignored

### DISCUSSION THREAD

**Discussion by arilebedey** [nix-community/kickstart-nix.nvim#25] [Q&A] [answered]
The [keymaps specified in telescope.lua](https://github.com/nix-community/kickstart-nix.nvim/blob/d1fb5431af40aff7a1646af7606920ea4bbb6d86/nvim/plugin/telescope.lua#L77) appear not to work.
Also when trying to add another keymap it doesn't get implemented. Here's what I tried adding:
```
local builtin = require('telescope.builtin')
vim.keymap.set('n', '<leader>sn', function()
  builtin.find_files { cwd = vim.fn.stdpath 'config' }
end, { desc = '[S]earch [N]eovim' })
```
Surely I'm missing something! 😄

**Comment by mrcjkb** [comment]
Hey 👋 

No idea what could be going on. 
They work fine for me. Is it just the telescope ones? 
Are you using the correct `<leader>` key?

**Reply by arilebedey** [reply]
I haven't done any setup. Just ran `nix run "github:nix-community/kickstart-nix.nvim"`.
The leader key has only 2 working shortcuts
![24-05-20_05-58-17](https://github.com/nix-community/kickstart-nix.nvim/assets/34816154/14a9ae2e-aaf8-4709-8192-5d03b0011aab)

**Reply by mrcjkb** [reply, marked as answer]
Neovim's default `<leader>` key isn't `<space>`, it's `\`.
See `:h leader`.
